packages/fxa-payments-server/src/components/PlanDetails/index.tsx

```diff
@@ -347,7 +347,8 @@ export const PlanDetails = ({
               </div>
 
               <div className="pt-4 pb-6">
-                {!!priceAmounts.totalAmount && (
+                {(priceAmounts.totalAmount === 0 ||
+                  !!priceAmounts.totalAmount) && (
```

Comment:
Q: If this is checking if `totalAmount` is 0 or truthy, do... do we need the check still?
Or in what cases would the total amount NOT be shown?

Reply:
Ha! Yes good point. `totalAmount` should always be shown. I've removed the check.